tests/test_stateful_browser.py

```diff
@@ -42,11 +46,11 @@ def test_submit_online():
     assert data["comments"] == "Some comment here"
     assert data["nosuchfield"] == "new value"
 
-    assert (json["headers"]["User-Agent"] ==
-            'testing https://github.com/hickford/MechanicalSoup')
+    assert (json["headers"]["User-Agent"] == 'testing MechanicalSoup')
```

Comment:
While you're there: unnecessary braces after assert (they were needed to allow the line break).